x-pack/plugins/secops/public/components/timeline/body/renderers/index.ts

```diff
@@ -31,7 +31,7 @@ export const columnRenderers: ColumnRenderer[] = [
 
 export const getSuricataCVEFromSignature = (signature: string): string | null => {
   const regex = /CVE-[0-9]*-[0-9]*/;
-  const found = signature.match(regex);
+  const found = (signature && signature.match(regex)) || null;
```

Comment:
probably should be `false` instead of ` || null` but still readable. I'll make notes for things like this.

Reply:
I can change that